examples/seismic/tti/tti_example.py

```diff
@@ -81,8 +83,15 @@ def test_tti_stability(shape, kernel):
                         help="Choice of finite-difference kernel")
     args = parser.parse_args()
 
-    # Switch to TTI kernel if input is acoustic kernel
-    preset = 'layers-tti-noazimuth' if args.azi else 'layers-tti'
+    # if user defines constant and noazimuth arguments at the same time raise exception
+    if args.constant and args.azi:
+        parser.error('argument --noazimuth: not allowed with argument --constant')
+
+    preset = 'layers-tti'
+    if args.constant:
+        preset = 'constant-tti'
+    elif args.azi:
+        preset = 'layers-tti-noazimuth'
```

Comment:
```
else:
    preset = 'layers-tii'
```

Reply:
Don't particularly like these additional presets. There is also no reason constant would not be compatible with no azimuth

Reply:
I wrote it in that way because there is not a` 'constant-noazimuth' `preset model. In any case you would have to play with conditionals I think